schema/test/config/bad/linux-netdevice.json

```diff
@@ -0,0 +1,13 @@
+{
+    "ociVersion": "1.0.0",
+    "root": {
+        "path": "rootfs"
+    },
+    "linux": {
+        "netDevices": {
+            "eth0": {
+                "name": 23
```

Comment:
what is 23 here?

Reply:
an integer to cause a bad config since it is expecting a string